libs/tracing/unix/Tracing.ml

```diff
@@ -83,7 +79,12 @@ let with_setup f =
    * ALT: we could also have wrapped this with a `Otel.Scope.with_ambient_scope`
      to ensure the trace_id is the same for all spans, but we decided that
      having the top level time is a good default. *)
-  Opentelemetry_client_ocurl.with_setup () @@ fun () ->
+  let config =
+    Some
+      (Opentelemetry_client_ocurl.Config.make
+         ?url:(Some "https://telemetry.dev2.semgrep.dev") ())
```

Comment:
What do you think about also having a flag for setting the endpoint too? I'm imagining this would default to prod (`https://telemetry.semgrep.dev`) if not specified, otherwise whatever the user passes as the value.

This would enable three use cases pretty cleanly:
1. Customers want to send us the trace, it goes to prod: `semgrep <scan|ci> --trace`
2. We want to send a trace for benchmarks, it goes to dev: `semgrep <scan|ci> --trace --trace-endpoint https://telemetry.dev2.semgrep.dev`
3. We want to send trace for developing locally, it goes to locally running trace collector (Jaeger): `semgrep <scan|ci> --trace --trace-endpoint localhost:4138`


I'm imagining we'd want customers to be able to `semgrep scan --trace` and have it show up in prod and

Reply:
I have an env var; would you prefer a flag?

Reply:
(Flags are just a bit annoying to add because you have to put them in both semgrep cli and osemgrep)

Reply:
An env var is totally fine, my feedback was really that we should have some mechanism for customizing the endpoint.